java/java-ranger-regression/alarm/Alarm_prop1.yml

```diff
@@ -0,0 +1,8 @@
+format_version: "2.0"
+input_files:
+  - ../../common/
+  - impl/
+  - prop1_True/
```

Comment:
Please don't put the verdict into directory or file names. It becomes difficult to track benchmarks when it turns out that verdicts are wrong and files/directories need to be renamed.

Reply:
Thanks Peter, I have removed the verdict from the file names.